src/components/Referrals/TradersStats.tsx

```diff
@@ -192,30 +196,31 @@ function TradersStats({ referralsData, traderTier, chainId, userReferralCodeStri
           </div>
         </Modal>
       </div>
-      {currentDiscountDistributions.length > 0 ? (
+      {currentDiscountDistributions.length > -1 ? (
```

Comment:
Wondering how the `length` could be not greater than -1?

Reply:
very strange indeed, set to 0 as it was